.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+    },
+  "postCreateCommand": "python3 -m venv .venv & source .venv/bin/activate & pip3 install --upgrade pip & pip3 install -r requirements.txt",
+  "postStartCommand": "mkdocs serve"
```

Comment:
Here we have a long running command that, again, gets the `postStartCommand` stuck. As per [the official documentation](https://code.visualstudio.com/remote/advancedcontainers/start-processes):

![image](https://github.com/eclipse/kura/assets/22748355/68298f27-27e8-4551-b447-c012e073da86)

Therefore I would suggest:

```suggestion
  "postStartCommand": "nohup bash -c 'mkdocs serve &'"
```

- `nohup` is a POSIX command which means "no hang up". Its purpose is to execute a command such that it ignores the [HUP](https://en.wikipedia.org/wiki/SIGHUP) (hangup) signal and therefore does not stop when the user logs out. In this particular case it means that it won't exit after 15 minutes
- `&` is used to run the process in background so that the `postStartCommand` exits and the devcontainer doesn't get stuck